app/src/main/java/com/x8bit/bitwarden/data/auth/repository/util/JwtTokenUtils.kt

```diff
@@ -27,7 +28,8 @@ fun parseJwtTokenDataOrNull(jwtToken: String): JwtTokenDataJson? {
 
     return try {
         json.decodeFromString<JwtTokenDataJson>(decodedDataJson)
-    } catch (_: Throwable) {
+    } catch (throwable: Throwable) {
+        Timber.e(throwable, "Failed to decode JwtTokenDataJson")
```

Comment:
I was thinking about also logging when the value fails to be base64 decoded. Thoughts on this?